src/Integration/WordPressEmails.php

```diff
@@ -186,6 +186,24 @@ public function disableSendConfirmationOnAdminEmail()
 		);
 	}
 
+	/**
+	 * Disables email on admin email address changed
+	 *
+	 * @action notification/init
+	 *
+	 * @since [Next]
+	 *
+	 * @return void
+	 */
+	public function disableSendConfirmationOnAdminEmailChanged()
+	{
+		if (getSetting('integration/emails/send_confirmation_on_admin_email_changed') !== 'true') {
+			return;
+		}
+
+		add_filter('send_site_admin_email_change_email', '__return_false');
+	}
```

Comment:
```suggestion
	/**
	 * Disables email on admin email address changed
	 *
	 * @filter send_site_admin_email_change_email
	 *
	 * @since [Next]
	 * @return bool
	 */
	public function disableSendConfirmationOnAdminEmailChanged(): bool
	{
		return getSetting('integration/emails/send_confirmation_on_admin_email_changed') !== 'true';
	}
```

Sorry for the chained review, but I just noticed how you could significantly improve and simplify this code